frankenphp.c

```diff
@@ -167,15 +167,11 @@ PHPAPI void get_full_env(zval *track_vars_array) {
     go_string key = full_env.r0[i * 2];
     go_string val = full_env.r0[i * 2 + 1];
 
-    // create PHP strings for key and value
-    zend_string *key_str = zend_string_init(key.data, key.len, 0);
+    // create PHP string for the value
     zend_string *val_str = zend_string_init(val.data, val.len, 0);
 
     // add to the associative array
-    add_assoc_str(track_vars_array, ZSTR_VAL(key_str), val_str);
-
-    // release the key string
-    zend_string_release(key_str);
+    add_assoc_str(track_vars_array, key.data, val_str);
```

Comment:
You can even use `add_assoc_str_ex` to avoid a strlen call, as I suppose that we length of the key would be the same as strlen on this as there won't be any NUL bytes. Or am I mistaken here?

Reply:
Alright. I pushed that now, as a separate commit (it's a habit so it makes it clearer on GitHub what has changed since last review/remark, I can adapt to what you're used as convention as well but this is what I do in php-src).
On approve I can either squash it for you and use the conventional commit format. Just let me know.

Reply:
We do the same thing, and we usually squash for you on merge.